docs/concepts/web3/nfts.md

```diff
@@ -0,0 +1,643 @@
+* In case of a Full Access key, the user should be redirected to the wallet to approve a transaction.
+
+A [JavaScript SDK](https://docs.near.org/docs/develop/front-end/near-api-js) exists to cover all of these scenarios. It has all of the necessary functionality to integrate Web/Node.JS applications with blockchain. This SDK is a perfect choice for the Web-based clients, but it’s not suitable for desktop or mobile based clients. Other libraries can be used for them:
+
+
+
+*  [.NET Client](https://github.com/good1101/NearClientApi/tree/master/NearClient) - suitable for Unity or Xamarin.
+* TODO - collect other libs here
+
+Same SDKs and libraries can be used for servers. The only difference is that a server cannot interact with a Wallet, so it must have access to a Full Access key, which should be stored and accessed in a secure way. 
+
+Also, another solution is available if a server uses a technology that doesn’t have NEAR SDK available for - we can create a separate (micro)service using the Node.js, which would handle all blockchain interactions:
+
+
+
+
+![image](/docs/assets/web3/nfts-21.png)
+
+
+
+
+An example of such a proxy server [can be found here](https://docs.near.org/docs/api/rest-server/overview).
+
+
+#### Contracts
+
+As we discovered in a previous section, for our application we need two smart contracts: for NFT and for Marketplace. There are two options on how to get them - use in-house implementation or some third-party/SAAS solution. Both options are viable and have different pros/cons. 
+
+If we want to create our own contract, we are fully in control and can implement anything we want. An obvious drawback, of course, is that it will take time and money to build it. Third-party solutions, on the other hand, are limited in their functionality and often cannot be easily extended. Also, they usually have some upfront costs and/or usage fees.
+
+For an in-house NFT contract implementation a few resources can be used as a starting point. First of all, a [Rust library](https://docs.rs/near-contract-standards/4.0.0-pre.6/near_contract_standards/index.html) is available which implements most of the standard. Another option is to build an entire contract from scratch, a good guide on how to do this is available by [this link](https://docs.near.org/docs/tutorials/contracts/nfts/introduction#).
+
+Implementing an own Marketplace contract is more involved since there is no standard implementation. A few examples:
+
+
+
+* Basic marketplace example - [https://docs.near.org/docs/tutorials/contracts/nfts/marketplace](https://docs.near.org/docs/tutorials/contracts/nfts/marketplace)
+* [Paras ](https://paras.id/)marketplace contract - [source](https://github.com/ParasHQ/paras-marketplace-contract/tree/master/paras-marketplace-contract/src). 
+* Marketplace example to buy NFTs using FTs - [source](https://github.com/near-examples/nft-tutorial/tree/8.marketplace).
```

Comment:
Remove last item. Originally linked repo is now dead, and currently linked one doesn't allow FTs purchase.